soc/BatteryPower.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+import sys
+import os.path
+
+class BatteryPower:
+    def __init__(self): 
```

Comment:
Might be quicker to just initialize resistance without the `__init__` since it's constant